src/test/tests/plots/molecule.py

```diff
@@ -0,0 +1,237 @@
+# ----------------------------------------------------------------------------
+#  CLASSES: nightly
+#
+#  Test Case:  molecule.py
+#
+#  Tests:      mesh      - 3D points
+#              plots     - Molecule
+#              operators - CreateBonds, Replicate
+#
+#  Programmer: Kathleen Biagas 
+#  Date:       June 15, 2021 
+#
+#  Modifications:
+#
+# ----------------------------------------------------------------------------
+
+def SetGradientBackground():
+    annot = GetAnnotationAttributes() 
+    annot.backgroundMode = annot.Gradient
+    annot.gradientBackgroundStyle = annot.Radial
+    annot.gradientColor1 = (102, 102, 153, 255)
+    annot.gradientColor2 = (0, 0, 0, 255)
+    annot.backgroundColor = (0, 0, 0, 255)
+    annot.foregroundColor = (255, 255, 255, 255)
+    SetAnnotationAttributes(annot)
+
+def SetWhiteBackground():
+    annot = GetAnnotationAttributes() 
+    annot.backgroundMode = annot.Solid
+    annot.foregroundColor = (0, 0, 0, 255)
+    annot.backgroundColor = (255, 255, 255, 255)
+    SetAnnotationAttributes(annot)
+
+def SaveTestImage(name):
```

Comment:
Hmm....a better name might be `Test500x500():` because it pairs better with `Test()`. I mean, even `Test()` saves a test image. What this is doing that is different is that it is saving a larger (500x500) image.

Reply:
Done